pkg/kubernetes/kubernetes_interface.go

```diff
@@ -115,4 +116,8 @@ type KubernetesConnector interface {
 	SetJWTToken(ctx context.Context, token string) error
 	// GetJWTToken returns the JWT token from the everest-jwt secret.
 	GetJWTToken(ctx context.Context) (string, error)
+	// UpdateEverestSettings accepts the full list of Everest settings and updates the settings.
+	UpdateEverestSettings(ctx context.Context, settings common.EverestSettings) error
```

Comment:
Considered an option to have Patch update for the settings - I think for now it's sufficiently to have only the full Update method. Not sure how the Settings are going to evolve and we can always add the Patch update later if we would need it.